enhancements/sig-policy/72-content-to-ansilbe-job/README.md

```diff
@@ -54,7 +56,9 @@ Current `extra_vars` has a field `target_clusters` contains non-compliant cluste
     "cluster1", 
     "cluster2", 
     "cluster3",
-  ]
+  ],
+  "job_template_name": "Demo Job Template",
+  "job_ttl": 86007,
```

Comment:
I don't think this in the extra vars. If it isn't, it's not supposed to be.

Reply:
I read the YAML output wrong